lib/docs/scrapers/pony.rb

```diff
@@ -0,0 +1,12 @@
+module Docs
+  class Pony < UrlScraper
+    self.type = 'pony'
+    self.release = '0.25.0'
+    self.base_url = 'https://stdlib.ponylang.io/'
+
+    html_filters.push 'pony/container', 'pony/clean_html', 'pony/entries'
+    options[:attribution] = "Me"
```

Comment:
Can you update this to include both the copyright and license information for the docs?

Reply:
Ow, how very sloppy of me. Will fix of course.